kolibri/core/content/api.py

```diff
@@ -291,6 +294,14 @@ def filter_available(self, queryset, name, value):
         return queryset.filter(root__available=value)
 
 
+class ChannelThumbnailView(View):
+    def get(self, request, channel_id):
+        channel = get_object_or_404(models.ChannelMetadata, id=channel_id)
+        thumbnail = channel.thumbnail
+        response = FileResponse(thumbnail)
```

Comment:
I am not confident this will be sufficient - the file is base64 encoded, which has a prefix with the mimetype. I think it is likely you would need to use the base64 decode method before serving this as the contents of a file: https://docs.python.org/3/library/base64.html#base64.b64decode